configs/sot/siamese_rpn/README.md

```diff
@@ -14,12 +14,23 @@
 }
 ```
 
-## Results and models on LaSOT dataset
+## Results and models on dataset
+
+### LaSOT
 
 We observe around 1.0 points fluctuations in Success and 1.5 points fluctuations in Norm percision. We provide the best model.
 
 Note that all of checkpoints from 11-th to 20-th epoch need to be evaluated in order to achieve the best results.
```

Comment:
`Note that the checkpoints from 10-th to 20-th epoch will be evaluated during training, you can find the best checkpoint from the log file`